renku_notebooks/api/classes/server.py

```diff
@@ -0,0 +1,399 @@
+import os
+import requests
+from kubernetes import client
+from kubernetes.client.rest import ApiException
+from hashlib import md5
+import base64
+import json
+from urllib.parse import urlparse
+from uuid import uuid4
+from flask import make_response, jsonify
+import logging
+from urllib.parse import urljoin
+
+
+from ...util.check_image import parse_image_name, get_docker_token, image_exists
+from ...util.kubernetes_ import (
+    get_k8s_client,
+    secret_exists,
+    filter_pods_by_annotations,
+)
+
+
+class Server:
```

Comment:
Just a naming suggestion, feel free to ignore... but casually glancing at this code I expected this to represent the jupyterhub server, not a user session. Maybe it could be called a `JupyterSession` or `UserSession` or `UserServer` or something similar?

Reply:
I see what you mean, I will update to UserServer.